main.go

```diff
@@ -73,19 +75,13 @@ func main() {
 		log.Fatalf("can't connect to database: %v\n", err)
 	}
 
-	// holds references for activated providers for current user token
-	scms := auth.NewScms()
-	bh := web.BaseHookOptions{
-		BaseURL: *baseURL,
-		Secret:  os.Getenv("WEBHOOK_SECRET"),
-	}
-
 	runner, err := ci.NewDockerCI(logger)
 	if err != nil {
 		log.Fatalf("failed to set up docker client: %v\n", err)
 	}
 	defer runner.Close()
 
+	// TODO(vera): find and replace (if possible) all occasions where this token is used
```

Comment:
We only use the token-based approach with `helpbot` and `approvelist`. It would be great to have the `cmd/approvelist` command line tool be updated as an example of how we would also have to update the `helpbot`.